src/sql/azdata.proposed.d.ts

```diff
@@ -5,6 +5,7 @@
 
 // This is the place for API experiments and proposal.
 
+import { URI } from 'vs/base/common/uri';
```

Comment:
You can't import stuff like this - you can only import from the vscode typings file (which has the URI definition I assume you're meaning to use)